.github/workflows/block-unconventional-commits.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow is provided via the organization template repository
+#
+# https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization
+
+name: Block unconventional commits
+
+on:
+  pull_request:
+    types: [opened, ready_for_review, reopened, synchronize]
+
+permissions:
+  contents: read
+
+concurrency:
+  group: block-unconventional-commits-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  block-unconventional-commits:
+    name: Block unconventional commits
+
+    runs-on: ubuntu-latest-low
```

Comment:
using `ubuntu-latets-low` makes it very unlikely that the action runs at all, as they usually just time out

Reply:
nvm